crates/libtiny_tui/src/tui.rs

```diff
@@ -626,6 +612,42 @@ impl TUI {
         tab.set_cursor(cursor);
     }
 
+    pub fn show_help_tab(&mut self, messages: &[String]) {
+        let tab_name = "help";
+        if let Some(tab_idx) = self.new_server_tab(tab_name, None) {
+            let target = &MsgTarget::Server { serv: tab_name };
+
+            let mut cmds = vec!["\x0303Commands: ".to_string()];
```

Comment:
Would be good to add a comment here saying that TUI supports IRC color codes and this is and `\x0303` part is IRC color code for ... color (green?).